integrations/access/msteams/msapi/config.go

```diff
@@ -30,10 +30,13 @@ type Config struct {
 	AppSecret string `toml:"app_secret"`
 	// TenantID ms tenant id
 	TenantID string `toml:"tenant_id"`
-	// Region bot framework api AP region
+	// Region to be used by the Microsoft Graph API client
 	Region string `toml:"region"`
 	// TeamsAppID represents Teams App ID
 	TeamsAppID string `toml:"teams_app_id"`
+	// DefaultRecipient is the default recipient to use if no access monitoring rules are specified.
+	DefaultRecipient string `toml:"default_recipient"`
+
```

Comment:
The default recipient is not part of the msteams api config, this is already implemebnted for self-hosted, we only need the fallback recipient in hosted mode. The slack plugin is implementing the default recipient in two ways:
- for self-hosted mode, we check that there's a '*' map entry when parsing the receipients field in the conf
- for hosted plugins we have a default route field in the [`e/lib/plugins/pluginConfiguration`](https://github.com/gravitational/teleport.e/blob/84f36a00ae432a6d8b08d6b84dd56a4970402bcf/lib/plugins/configuration.go#L14), and we populate the field in the instance factory: https://github.com/gravitational/teleport.e/blob/84f36a00ae432a6d8b08d6b84dd56a4970402bcf/lib/plugins/slack.go#L28

Can you remove this from the tems config and have this plugin use the common plugin config?

Reply:
I have updated the [/e PR](https://github.com/gravitational/teleport.e/pull/5032/) to populate the recipients map in the common baseconfig.